sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3590,6 +3873,35 @@ public partial class PredictiveValue : System.ClientModel.Primitives.IJsonModel<
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PredictiveValue>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PredictiveValue>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Processor : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Processor>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Processor>
```

Comment:
Single word name is not allowed, could we specify the name here?

Reply:
Renamed to `PipelineGroupProcessor`